R/dvhydrograph-data.R

```diff
@@ -103,6 +105,21 @@ parseRefData <- function(data, series) {
   return(plotData)
 }
 
+parseEstimatedEdges <- function(data, allVars){
```

Comment:
I don't think you need to pass in `data`. You don't use it and then overwrite it on line 115 anyways :)